budcamp/Homework3/Homework3/task1/Lithuania.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework3
+{
+    internal class Lithuania: History, ICountry
```

Comment:
I agree with you that inheritance make sense for types `Lithuania` and `ICountry` as `Lithuania` **is a** `Country` but it doesn't make sense for `History`.
Please use composition instead for `History`.